src/murmur/DBus.h

```diff
@@ -103,6 +103,12 @@ struct LogEntry {
 Q_DECLARE_METATYPE(LogEntry);
 Q_DECLARE_METATYPE(QList<LogEntry>);
 
+class MurmurDBusConnectionHolder {
```

Comment:
I'm not quite seeing the point of MurmurDBusConnectionHolder. The way it is written it seems we can copy construct QDBusConnection so why not use a QDBusConnection* directly? If we do not care about the precise timing of the cleanup we can make it a smart pointer and don't have to care about cleanup either. Am I missing something?